contracts/tokens/Cruzo1155.sol

```diff
@@ -147,4 +149,36 @@ contract Cruzo1155 is Initializable, ERC1155URI {
     function setContractURI(string memory _newURI) external onlyOwner {
         contractURI = _newURI;
     }
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        override(ERC2981Upgradeable, ERC1155Upgradeable)
+        returns (bool)
+    {
+        return super.supportsInterface(interfaceId);
+    }
+
+    function setDefaultRoyaltyInfo(address _receiver, uint96 _royaltyFeesInBips)
+        public
+        onlyOwner
+    {
+        require(
+            _royaltyFeesInBips <= 5000,
+            "Royalty value must be between 0% and 50%"
+        );
+        _setDefaultRoyalty(_receiver, _royaltyFeesInBips);
+    }
+
+    function setTokenRoyalty(
+        address _receiver,
+        uint96 _royaltyFeesInBips,
+        uint256 _tokenId
+    ) public onlyOwner {
```

Comment:
I'm not sure about `onlyOwner`. It'll work for collections which are created and owned by users. This way users will not have a possibility to control royalties when creating tokens in the default Cruzo collection, which is owned by us. Should not we use `onlyCreator` instead?